### PR TITLE
Created framework and added few testsc

### PR DESCRIPTION
**Pull request by antonina12**

### CODE REVIEW

File: test/pageobjects/dashboard.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+const Page = require('./page');
+
+class DashboardPage extends Page {
+    get username () {
+        return $('#buzz-content');
+    }
+
+    get accountMenu () {
+        //return $('//div[contains(@class,"bellContainer")]');
```
Comment (vladilchenko) [inline test/pageobjects/dashboard.js:9]: Forgot to remove comment

File: test/pageobjects/quizzes_page.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+const Page = require('./page');
+
+class QuizzePage extends Page {
+    get quiz_cat_filter () {
```
Comment (vladilchenko) [inline test/pageobjects/quizzes_page.js:4]: What is cat ?

File: test/specs/account_menu.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+const LoginPage = require('../pageobjects/login.page');
+const DashboardPage = require('../pageobjects/dashboard');
+
+describe('Account menu', () => {
+    it('check menu items', async () => {
+        LoginPage.open('');
```
Comment (vladilchenko) [inline test/specs/account_menu.js:6]: remove string argument

File: test/pageobjects/quizzes_page.js (hunk 2 above)
Comment (antonina12) [reply test/pageobjects/quizzes_page.js:4]: This is short name for 'category'

File: test/specs/login.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+const LoginPage = require('../pageobjects/login.page');
+const DashboardPage = require('../pageobjects/dashboard');
+
+describe('My Login application', () => {
+    it('should login with valid credentials', async () => {
+        LoginPage.open('');
```
Comment (vladilchenko) [inline test/specs/login.js:6]: remove string argument

File: test/pageobjects/quizzes_page.js (hunk 2 above)
Comment (antonina12) [reply test/pageobjects/quizzes_page.js:4]: > What is cat ?

This is short name for 'category'

**Review by vladilchenko — CHANGES_REQUESTED** [review]
please fix all comments

File: test/pageobjects/dashboard.js (hunk 1 above)
Comment (antonina12) [reply test/pageobjects/dashboard.js:9]: > Forgot to remove comment

I want this comment for my future testcases. Please, don't be so mad

**Review by vladilchenko — APPROVED** [review]

**Pull request merged**